test/common.py

```diff
@@ -2340,6 +2340,11 @@ def browser_open(cls, url):
     if not EMTEST_BROWSER:
       logger.info('No EMTEST_BROWSER set. Defaulting to `google-chrome`')
       EMTEST_BROWSER = 'google-chrome'
+    if WINDOWS:
+      # On Windows env. vars canonically use backslashes as directory delimiters, e.g.
+      # set EMTEST_BROWSER=C:\Program Files\Mozilla Firefox\firefox.exe
+      # and spaces are not escaped, so convert backslashes to forward slashes to feed to shlex split.
```

Comment:
This doesn't handle the space problem does it?    I think would would need to use quotes or escape the spaces in this case.